package.json

```diff
@@ -10,7 +10,8 @@
   ],
   "main": "lib/index.js",
   "scripts": {
-    "test": "make test"
+    "test": "make test",
+    "example": "beefy example.js"
```

Comment:
This is an interesting idea 🤔 

We're working on overhauling documentation, would be neat to have code examples in the readme on how to get this set up using the ajs open source. But for now, in keeping w our existing testing/documentation infra, could we remove this line as well as the `example.js` file?